content/source/docs/cloud/run/ui.html.md

```diff
@@ -86,6 +86,8 @@ Whenever a pull request is _created or updated,_ Terraform Cloud checks whether
 
     -> **Note:** If Terraform Cloud skips a plan because the changes weren't relevant, it will still post a passing commit status to the pull request.
 
+- Status checks on the pull request are not updated after an apply, so a commit with a successful plan but an errored apply will still show the passing commit status relevant to the plan.
```

Comment:
Just a suggestion to remove some passive voice, and a clarification. Is the apply we're talking about here the apply that results from merging the PR itself?

```suggestion
- Terraform Cloud does not update the status checks on the pull request with the status of the associated apply. A commit with a successful plan but an errored apply will still show the passing commit status from the plan.
```

Reply:
> Just a suggestion to remove some passive voice, and a clarification. Is the apply we're talking about here the apply that results from merging the PR itself?

Yes! That's correct. I'm going to propose another version of this that merges both you and Nick's comments:

```suggestion
- Terraform Cloud does not update the status checks on a pull request with the status of an associated apply. This means that a commit with a successful plan but an errored apply will still show the passing commit status from the plan.
```

Sound good @nfagerlund @judithpatudith ?